Mg write procedures doc (#216)
memgraph/docs: docs/database-functionalities/query-modules/implement-query-modules.md

                  avg_degree=avg_degree, props=props)
```

### Writeable procedure

> Writeable procedures can be implemented in very similar way as read-only

Reviewer: ```suggestion
Writeable procedures can be implemented in a very similar way as read-only
```

> Writeable procedures can be implemented in very similar way as read-only

Reviewer 2: ```suggestion
Writeable procedures can be implemented in a very similar way as read-only
```

Author: @jbajic had the same comment, already applied his changes.

procedures. The only difference is writeable procedures receive mutable objects,

> therefore they can create or delete vertices or edges, modify the properties of

Reviewer 2: ```suggestion
therefore they can create and delete vertices or edges, modify the properties of
```

vertices and edges and they can add or remove labels of vertices.

> We can implement a very simple writeable query modules similarly to read-only

Reviewer 2: ```suggestion
We can implement a very simple writeable query module similarly to read-only
```

> We can implement a very simple writeable query modules similarly to read-only
> procedures:

Reviewer: ```suggestion
We can implement a very simple writeable query module similarly to read-only
procedures:
```

```python
@mgp.write_proc
def write_procedure(context: mgp.ProcCtx,
                    property_name: str,
                    property_value: mgp.Nullable[mgp.Any]
                    ) -> mgp.Record(created_vertex=mgp.Vertex):
    # Collect all the vertices that has the required property with the same
    # value
    vertices_to_connect = []
    for v in context.graph.vertices:
        if v.properties[property_name] == property_value:
            vertices_to_connect.append(v)
    # Create the new vertex and set its property
    vertex = context.graph.create_vertex()
    vertex.properties.set(property_name, property_value)
    # Connect the new vertex to the other vertices
    for v in vertices_to_connect:
        context.graph.create_edge(vertex, v, mgp.EdgeType("HAS_SAME_VALUE"))

    return mgp.Record(created_vertex=vertex)

```

This example procedure creates a new vertex with the specified property and

> connects it to all existing vertex which has the same property with the same

Reviewer 2: ```suggestion
connects it to all existing vertices which have the same property with the same
```

Author: @jbajic had the same comment, already applied his changes.
